prow/github/client.go

```diff
@@ -519,6 +521,82 @@ func (c *Client) UnassignIssue(org, repo string, number int, logins []string) er
 	return nil
 }
 
+func (c *Client) tryRequestPRReview(org, repo string, number int, logins []string) (int, error) {
+	c.log("RequestPRReview", org, repo, number, logins)
+	var pr PullRequest
+	return c.request(&request{
+		method:      http.MethodPost,
+		path:        fmt.Sprintf("%s/repos/%s/%s/pulls/%d/requested_reviewers", c.base, org, repo, number),
+		requestBody: map[string][]string{"reviewers": logins},
+		exitCodes:   []int{201},
+	}, &pr)
+}
+
+// RequestPRReview tries to add the users listed in 'logins' as requested reviewers of the specified PR.
```

Comment:
PR Review Requests are still in the "developer preview" window, and I'm guessing this is the sort of behavior that will be broken at some point.